op-node/rollup/driver/state.go

```diff
@@ -30,8 +30,8 @@ type Driver struct {
 	// When the derivation pipeline is waiting for new data to do anything
 	idleDerivation bool
 
-	// Requests for sync status. Synchronized with event loop to avoid reading an inconsistent sync status.
-	syncStatusReq chan chan eth.SyncStatus
+	// Requests for synchronized event loop execution to avoid reading an inconsistent state
```

Comment:
Better comment would be `Requests to block the event loop`. There's no execution inside this channel, but it's instead holding the event loop for external execution (aka a mutex)